procedures/lvv-t1992.ipynb

```diff
@@ -20,239 +20,277 @@
    "source": [
     "from lsst.ts import salobj\n",
-    "from lsst.ts.idl.enums import Hexapod\n",
+    "from lsst.ts.idl.enums import MTHexapod\n",
     "import logging\n",
     "import asyncio\n",
-    "import os"
+    "import os\n",
+    "import yaml\n",
+    "\n",
+    "import numpy as np\n",
+    "from matplotlib import pyplot as plt\n",
+    "from astropy.time import Time\n",
+    "from datetime import datetime, timedelta\n",
+    "import pandas as pd\n",
+    "import astropy.units as u"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following block sets the necessary environment variables for setting up the DDS/SAL communication"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
-    "os.environ[\"LSST_DDS_DOMAIN\"] = 'citest'\n",
-    "os.environ[\"OSPL_URI\"] = \"file:///scratch/tribeiro/ospl.xml\"\n",
+    "#os.environ[\"LSST_DDS_DOMAIN\"] = 'citest'\n",
     "os.environ[\"LSST_DDS_HISTORYSYNC\"] = \"10\""
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
-    "STD_WAIT = 39"
+    "STD_WAIT = 39 # This is the amount of time to wait in between a move just to make sure that the actuators are not overheating"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
-    "log = logging.getLogger(__name__)"
+    "log = logging.getLogger(__name__) # This sets up the logger for the test."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test_message = \"Camera Hexapod Integration Test\"\n",
+    "script = salobj.Controller(\"Script\", index=42658885)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
    "metadata": {},
+   "source": [
+    "This is how you start the remote for the CSC."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [
     {
-     "name": "stderr",
+     "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Could not read historical data in 10.20 sec\n"
+      "<lsst.ts.salobj.domain.Domain object at 0x7f1564382dc0>\n",
+      "<lsst.ts.salobj.remote.Remote object at 0x7f1564382b50>\n"
      ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[<State.STANDBY: 5>, <State.DISABLED: 1>, <State.ENABLED: 2>]"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
     "domain = salobj.Domain()\n",
-    "hexapod_csc = salobj.Remote(name=\"Hexapod\", domain=domain, index=1)\n",
-    "await hexapod_csc.start_task"
+    "print (domain)\n",
+    "#index=1 is the camera hexapod!! index=2 is the M2 hexapod!\n",
```

Comment:
Please use the SalIndex enum

Reply:
Updated for the latest notebook, that I will use now. Here tbd if used again!